library/ui-strings/src/main/res/values/strings.xml

```diff
@@ -3346,6 +3346,8 @@
     <string name="labs_enable_session_manager_summary">Have greater visibility and control over all your sessions.</string>
     <string name="labs_enable_client_info_recording_title">Enable client info recording</string>
     <string name="labs_enable_client_info_recording_summary">Record the client name, version, and url to recognise sessions more easily in session manager.</string>
+    <string name="labs_enable_voice_broadcast_title">Enable Voice broadcast (under active development)</string>
+    <string name="labs_enable_voice_broadcast_summary">Be able to record and send voice broadcast in room timeline. Only ended voice broadcast can be listened (listen live broadcast is not correctly supported for the moment)</string>
```

Comment:
@Florian14 please replace this wording with "Be able to record and send voice broadcast in room timeline."